### ISSUE TITLE
Revving static files

### ISSUE THREAD

**Issue by S1SYPHOS** [ixkaito/frasco#15]
Hey there,
would you like for static assets to be revisioned? Revving them, writing stuff to a `manifest.json`, maybe in Jekyll's `_data` folder, then referencing them, you know?

**Comment by ixkaito** [comment]
@S1SYPHOS 

Thanks for your idea! The more simple and flexible structure would be better. Do you think this is?

**Comment by S1SYPHOS** [comment]
@ixkaito Totally, but it's your call - I'll provide an idea so you may see for yourself!

**Comment by ixkaito** [comment]
@S1SYPHOS Thanks! I'll also check by myself!

**Comment by S1SYPHOS** [comment]
I'd like to separate source files (everything going into the site) and config/node files (node_modules, package.json, gulpfile.js etc stay at root level), so the sctructure be 

```
├── app
│   ├── _includes
│   ├── _posts
│   ├── _layouts
│   ├── ...
│   └── index.html
│
├── dist (renamed _site)
│   └── ...
│
├── package.json
├── gulpfile.js
├── frasco.webpack.config
├── ...
```

just a thought :v:

**Comment by ixkaito** [comment]
@S1SYPHOS 

Thanks! I know this will make the structure more clear, but I want this project to be available for GitHub Pages simply to push it. Also, for Jekyll beginners, I think it's better to have the default Jekyll structure, e.g. `_layouts`, `_includes`, `_posts`, `_site` etc.

**Comment by S1SYPHOS** [comment]
I totally understand this, thanks

**Comment by ixkaito** [comment]
@S1SYPHOS Released v1.0.0 but didn't change the file structure. Thank you for your issue again.
